internal/provider_test/integration_test.go

```diff
@@ -21,8 +22,53 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func IsE2E() bool {
-	return os.Getenv("E2E_TEST") == "true"
```

Comment:
We'll need to change how `nuodb-dbaas-deploy` executes the TF plugin tests.

Reply:
Thanks for the reminder.